### PR TITLE
Implemented selected modpack header

### PR DESCRIPTION
**Pull request by mattie078**
app/landing.ejs - line 170
`					<pre style="text-align: center;white-space: normal;">
`

While it normally isn't good practice to have CSS in the style tag while having a CSS file. I still find it reasonable to exclude preformatted text. Due to the nature of that tag.

### CODE REVIEW

File: app/landing.ejs [hunk 1]
```diff
@@ -162,11 +162,20 @@
...
-					<div class="bot_divider"></div>
-					<button id="server_selection_button" class="bot_label">&#8226; No Server Selected</button>
+					<div class="bot_divider_right"></div>
+					
+					<pre style="text-align: center;white-space: normal;">
+					
+						<span class="bot_selected_modpack">Selected modpack:</span>
+						<br>
+						<br>
+						<button id="MM_server_selection_button">&#8226; No Server Selected</button>
```
Comment (Brend-Smits) [inline app/landing.ejs]: No server or no modpack selected?

Comment (Brend-Smits) [reply app/landing.ejs]: I would also prefer it if id's and classes were all lower case.

**Review by Brend-Smits — CHANGES_REQUESTED** [review]

**Comment by mattie078** [comment]
Now that I finished my internship I will be having more time to also fix some issues with the current launcher.

**Comment by Brend-Smits** [comment]
I will check this out today. Thanks for assigning me :)

**Comment by Brend-Smits** [comment]
Please note that there is still a conflict.

**Review by Brend-Smits — APPROVED** [review]
Not tested, but I assume you did test it. Code looks great, thanks!

**Pull request merged**